juniper/src/lib.rs

```diff
@@ -33,8 +33,8 @@ mod ast;
 pub mod executor;
 mod introspection;
 pub mod parser;
-pub(crate) mod schema;
-mod types;
```

Comment:
@tusharmath here types are mostly re-exported to the crate's root. If something is missing, I think it's better to add that to the re-exports, rather than making these modules public.